app/lib/import_export_constants.rb

```diff
@@ -522,7 +523,7 @@ def self.rct_node(schema, label, fields)
   HISTORIES_EXPORT_OPTIONS = {
     label: 'Histories',
     nodes: [rct_node(:histories, 'History', %i[patient_id user_defined_id_statelocal user_defined_id_cdc user_defined_id_nndss id created_by history_type
-                                               comment created_at updated_at])]
+                                               comment created_at updated_at was_edited])]
```

Comment:
let's move `was_edited` to be before the timestamps, same for line 378

Reply:
done here: https://github.com/SaraAlert/SaraAlert/pull/793/commits/ad9df046c5344445c1cf056f8dfb316aee18461a

Reply:
also now obsolete